init/fluent_bit_init_process.go

```diff
@@ -0,0 +1,401 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/sirupsen/logrus"
+)
+
+// static paths
+const (
+	s3FileFolderPath       = "/init/fluent-bit-init-s3-files/"
+	mainConfigFile         = "/init/fluent-bit-init.conf"
+	originalMainConfigFile = "/fluent-bit/etc/fluent-bit.conf"
+	invokeFile             = "/init/invoke_fluent_bit.sh"
+)
+
+var (
+	// default Fluent Bit command
+	fluentBitCommand = "exec /fluent-bit/bin/fluent-bit -e /fluent-bit/firehose.so -e /fluent-bit/cloudwatch.so -e /fluent-bit/kinesis.so"
+
+	// global s3 client and flag
+	s3Client      *s3.S3
+	exists3Client bool = false
```

Comment:
nit: `existS3Client`

Reply:
Actually, `s3ClientExists` or `s3ClientCreated` would be more idiomatic I think... this fits what I've seem from other go programmers, if you have a bool, you use works like has or is or something else, but you put it in the variable name in a way that fits how you'd say it out loud in english, so `hass3Client` but that sounds weird to me... I prefer `s3ClientCreated` since its readable is exactly what you mean- has the S3 client been created or not yet?

Reply:
Already modified this variable name. Thanks for advice